examples/dynamo/eg6/omp_script.py

```diff
@@ -22,5 +22,6 @@ def trans(psy):
 
     # take a look at what we've done
     schedule.view()
+    schedule.dag(file_format="png")
```

Comment:
I've uninstalled the graphviz python bindings and successfully run the examples and the tests. Great.